init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/model/IDResponse.java

```diff
@@ -0,0 +1,11 @@
+package uk.ac.ox.ndph.mts.init_service.model;
```

Comment:
Is this class really necessary? It's just encapsulating a string really.

Reply:
It's for deserializing the bit of json returned by site-service and practitioner-service following a successful create.   (NB role service returns the whole role).    webclient has a convenient  .bodyToMono([class])  method which I'm using here sendBlockingPostRequest method here https://github.com/NDPH-ARTS/mts-services/blob/c62f75849f7f38b4a03685fbd4c19bda35b77e77/init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/service/ServiceInvoker.java and passing it this model.  Could possibly do the same with an anon inner class or something, but not sure it would be clearer?